CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(metalang99)
```

Comment:
```suggestion
project(Metalang99)
```

Reply:
This is down to personal preference, but from what I've seen most libraries keep this lowercase. Up to you, though!

Reply:
Oh, I also did it that way to match the repo name. That can cause some confusion, sometimes, if they don't match.

Reply:
Fair. IIRC, CMake project names are case-insensitive anyway.